samples/dataQuery/command.js

```diff
@@ -0,0 +1,54 @@
+const Superagent = require('superagent');
+const mysql = require('promise-mysql');
+/**
+* This is the implementation of your Fusebot command.
+* You can run it from Slack with `/fusebot run-query ...`.
+* Fusebot is powered by the Fusebit integration platform, https://fusebit.io/
+* 
+* Docs and samples are at https://github.com/fusebit/fusebot
+* You can talk to a fellow developer on Slack at https://fusebit.io/contact
+* Or tell us what you think with `/fusebot feedback {your-comments}` 
+*
+* Now let's build something already. 
+
+* @param ctx {FusebotContext}
+*/
+module.exports = async (ctx) => {
+    if (ctx.body.args.length === 0 || ctx.body.args[0] === "help") {
+        return help(ctx)
+    }
+    if (!ctx.configuration.HOST || !ctx.configuration.PORT || !ctx.configuration.USERNAME || !ctx.configuration.PASSWORD) {
+        return ctx.client.sendEphemeral("Unable to detect SQL connection settings from configuration. For more information, please check https://github.com/fusebit/fusebot/tree/main/samples/dataQuery")
+    }
+    const sqlStatement = ctx.body.args.join(" ")
+    try {
+        const conn = await mysql.createConnection({
+            host: ctx.configuration.HOST,
+            port: ctx.configuration.PORT,
+            user: ctx.configuration.USERNAME,
+            password: ctx.configuration.PASSWORD
+        })
+        const results = await conn.query(sqlStatement)
+        for (const row of results) {
+            console.log(row)
+            ctx.client.send(JSON.stringify(row))
```

Comment:
Why not just blast JSON.stringify(results) in one call back to slack? Also consider trimming the results to say 20 records, and if there are more say something at the bottom like "... [X] more matching results. Please limit your query". We don't want folks dumping entire databases in to Slack.

Reply:
Also, add `await` in front of all those sends across in this file.

Reply:
I feel iterating through the results would be nicer for reading n stuff. the rest are done.